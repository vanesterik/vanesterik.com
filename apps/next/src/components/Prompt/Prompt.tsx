import { text } from '@vanesterik/ui'

export const Prompt = () => (
  <div className={text({ intent: 'label' })}>&gt;&gt;&gt;</div>
)
