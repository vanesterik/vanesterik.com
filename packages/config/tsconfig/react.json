{
  "$schema": "https://json.schemastore.org/tsconfig",
  "display": "React",
  "extends": "./base.json",
  "compilerOptions": {
    "jsx": "react-jsx",
    "lib": ["dom", "dom.iterable", "ES2015"],
    "module": "ESNext",
    "resolveJsonModule": true,
    "target": "ES6"
  }
}
