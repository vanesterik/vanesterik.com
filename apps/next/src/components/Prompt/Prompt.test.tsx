import { render } from '@testing-library/react'
import { describe, expect, it } from 'vitest'

import { Prompt } from './Prompt'

describe('Prompt', () => {
  // @todo Refactor to use `getByText` instead of `container`
  it('should render as expected', () => {
    const { container } = render(<Prompt />)
    expect(container).toMatchSnapshot()
  })
})
