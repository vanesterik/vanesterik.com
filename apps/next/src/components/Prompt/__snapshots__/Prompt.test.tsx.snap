// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Prompt > should render as expected 1`] = `
<div>
  <div
    class="font-mono font-normal text-black text-xs uppercase dark:text-white"
  >
    &gt;&gt;&gt;
  </div>
</div>
`;
